packages/core/src/test/testAuthUtil.ts

```diff
@@ -10,6 +10,9 @@ import { LanguageClientAuth } from '../auth/auth2'
 import { AuthUtil } from '../codewhisperer/util/authUtil'
 
 export async function createTestAuthUtil() {
+    sinon.restore()
+    AuthUtil.destroy()
```

Comment:
Can you move this right before the Authutil.create() below and then add a comment why we need to call destroy right before create